### PR TITLE
Accept goverance on old contracts

### PR DESCRIPTION
**Pull request by DanielVF**

### CODE REVIEW

File: brownie/runlogs/2022_10_strategist.py [hunk 1]
```diff
@@ -194,4 +194,61 @@
...
+
+old_contracts = [
+  "0x9C94df9d594BA1eb94430C006c269C314B1A8281",  # CompensationClaims
+  "0xD5433168Ed0B1F7714819646606DB509D9d8EC1f",  # CompoundStrategyProxy
+  "0x9f2b18751376cF6a3432eb158Ba5F9b1AbD2F7ce",  # InitializeGovernedUpgradeabilityProxy
+  "0x3c5fe0a3922777343CBD67D3732FCdc9f2Fa6f2F",  # ThreePoolStrategyProxy
+  "0x48Cf14DeA2f5dD31c57218877195913412D3278A",  # VaultCore
+  "0x12115A32a19e4994C2BA4A5437C22CEf5ABb59C3",  # CompoundStrategyProxy
+  "0x47211B1D1F6Da45aaEE06f877266E072Cf8BaA74",  # CompoundStrategyProxy
+  "0x051CaEFA90aDf261B8E8200920C83778b7B176B6",  # InitializeGovernedUpgradeabilityProxy
```
Comment (sparrowDom) [inline brownie/runlogs/2022_10_strategist.py:216]: this one holds Aave strategy implementation

File: brownie/runlogs/2022_10_strategist.py [hunk 2]
```diff
@@ -194,4 +194,61 @@
...
+# 
+
+from world import *
+
+newgov = Contract.from_explorer("0x3cdD07c16614059e66344a7b579DAB4f9516C0b6")
+
+old_contracts = [
+  "0x9C94df9d594BA1eb94430C006c269C314B1A8281",  # CompensationClaims
+  "0xD5433168Ed0B1F7714819646606DB509D9d8EC1f",  # CompoundStrategyProxy
+  "0x9f2b18751376cF6a3432eb158Ba5F9b1AbD2F7ce",  # InitializeGovernedUpgradeabilityProxy
```
Comment (sparrowDom) [inline brownie/runlogs/2022_10_strategist.py:211]: Also holds AaveStrategy implementation

File: brownie/runlogs/2022_10_strategist.py [hunk 3]
```diff
@@ -194,4 +194,61 @@
...
+		print(contract.governor())
+
+
+newgov.propose(
+	[x.receiver for x in txs],
+	[0 for x in txs],
+	['claimGovernance()' for x in txs],
+	[x.input[10:] for x in txs],
+	"Claim governance on old contracts\n\nMoves a selection of old contracts from old goverance to the new.",
+	{'from': GOV_MULTISIG }
```
Comment (sparrowDom) [inline brownie/runlogs/2022_10_strategist.py:237]: Souldn't this be `OLD_GOV_MULTI_SIG` ?

File: brownie/runlogs/2022_10_strategist.py [hunk 4]
```diff
@@ -194,4 +194,61 @@
...
+txs = []
+
+with TemporaryFork():
+	for address in old_contracts:
+		contract = Contract.from_explorer(address)
+		txs.append(contract.claimGovernance({'from': newgov.timelock()}))
+		print(contract.governor())
+
+
+newgov.propose(
```
Comment (sparrowDom) [inline brownie/runlogs/2022_10_strategist.py:231]: and this `oldgov`

**Comment by sparrowDom** [comment]
Left a couple of comments

File: brownie/runlogs/2022_10_strategist.py (hunk 2 above)
Comment (DanielVF) [reply brownie/runlogs/2022_10_strategist.py]: Should be done.

File: brownie/runlogs/2022_10_strategist.py (hunk 1 above)
Comment (DanielVF) [reply brownie/runlogs/2022_10_strategist.py]: Should be done.

File: brownie/runlogs/2022_10_strategist.py (hunk 3 above)
Comment (DanielVF) [reply brownie/runlogs/2022_10_strategist.py:237]: See previous note

File: brownie/runlogs/2022_10_strategist.py (hunk 4 above)
Comment (sparrowDom) [reply brownie/runlogs/2022_10_strategist.py:231]: oh ok got it

**Review by sparrowDom — APPROVED** [review]
lgtm

**Pull request merged**